formatters/html/html.go

```diff
@@ -28,6 +29,11 @@ func TabWidth(width int) Option { return func(f *Formatter) { f.tabWidth = width
 // PreventSurroundingPre prevents the surrounding pre tags around the generated code
 func PreventSurroundingPre() Option { return func(f *Formatter) { f.preventSurroundingPre = true } }
 
+// Add custom user stylesheet
+func UserStylesheet(stylesheet string) Option {
```

Comment:
I'd prefer this to be the raw CSS rather than a file name, as it forces a specific location for the stylesheet. Also rename this to EmbedUserStylesheet or InlineUserStylesheet to make it clear what it does.